xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-ui/src/main/resources/CKEditor/Translations.fr.xml

```diff
@@ -59,7 +59,7 @@ ckeditor.plugin.link.queryString=Paramètres d'URL
 ckeditor.plugin.link.defaultLabel=Saisir le label du lien
 ckeditor.plugin.link.openInNewTab=Ouvrir le lien dans un nouvel onglet
 ckeditor.plugin.list.listType.none=Aucun
-ckeditor.plugin.macro.placeholder=macro : {0}
+ckeditor.plugin.macro.placeholder=macro:{0}
```

Comment:
@manuelleduc why this translation change? In french there's two spaces before and after double point characters (e.g. `;`, `:`, `!` etc)

Reply:
My bad, this change can be reverted.